docs/development.rst

```diff
@@ -8,6 +8,29 @@ Quickstart to run MapEntity in development :
     $ docker-compose run --rm web ./manage.py createsuperuser
     $ docker-compose up
 
+Develop with docker::
+
+    $ docker-compose up
+
+Get Screamshoter and convertit working:
+
+To get screamshotter container working, it needs to access to your web container by using your browser url.
```

Comment:
To make the screenshotter container work, it needs to access your web container by using your browser's URL.